analyses/oncoprint-landscape/00-map-to-sample_id.R

```diff
@@ -185,10 +185,36 @@ readr::write_tsv(maf_df, maf_output)
 
 message("Preparing fusion file...")
 
-# Separate fusion gene partners
-fus_sep <- fusion_df %>%
+# Sort fusions first in alphabetical order
+fus_sort <- fusion_df %>%
+  tibble::rowid_to_column("Fusion_ID") %>%
+  group_by(Sample, FusionName, Fusion_ID) %>%
+  mutate(SortedFusionName = paste(sort(unlist(strsplit(FusionName, "--"))), collapse = "--")) %>%
+  ungroup()
+
+# Now let's perform some filtering on our sorted data frame
+fus_filter <- fus_sort %>%
+  group_by(Sample, SortedFusionName) %>%
+  count() %>%
+  # Filter out redundant instances that are not true multi hit fusions
+  filter(n <= 1)
```

Comment:
Here, you can select out the columns you need for `fus_sep` to shorten this df.

Reply:
This df was removed altogether in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1009/commits/b640f5d9d133e004b37b84a306cffd92b216423e so I am going to resolve this conversation.